manage.py

```diff
@@ -188,3 +189,20 @@ def translation_download(language, state):
     state = state or default_state
     click.echo('Downloading {state} translations from Smartling'.format(state=state))
     smartling_download(state=state, language=language)
+
+
+@click.option('--config_key', '-c', help='Return only a single config value')
+@app.cli.command()
+def config(config_key):
+    """List current flask configuration values in JSON"""
+
+    if config_key:
+        # Remap None values to an empty string
+        print(app.config.get(config_key, '') or '')
```

Comment:
the or here is unnecessary.  the second param to `get` is the default if not found.

Reply:
If I don't have the `or`, config values that defined, but `None`(see below), will print `None` (instead of no printing)
```
config = {
    "key": None
}
```